admin/actions.php

```diff
@@ -728,3 +744,37 @@ function hmbkp_ajax_cron_test() {
 
 }
 add_action( 'wp_ajax_hmbkp_cron_test', 'hmbkp_ajax_cron_test' );
+
+/**
+ * Spawn a request to `wp-cron.php` and return the response.
+ *
+ * This function is designed to mimic the functionality in `spawn_cron()` with the addition of returning
+ * the result of the `wp_remote_post()` request.
+ *
+ * @return WP_Error|array The response or WP_Error on failure.
+ */
+function hmbkp_get_cron_spawn() {
```

Comment:
I'm not clear why we need this function instead of just the `wp_remote_head` requests we were doing inline at https://github.com/humanmade/backupwordpress/pull/822/files#diff-b2d8a71a75e0cf40115b3529a31e0761L702 seems like it introduces a lot of new logic / filtering options that could impact how this works?